Automattic/newspack-plugin
# Setup Wizard: Settings step

### All Submissions:






### Changes proposed in this Pull Request:



Settings step of the Setup Wizard.

Removes the `_newspack_profile_*` options, since they were not used anywhere anymore. Instead, WooCommerce `countrystate` & `currency` options are used. For this reason, WooCommerce is added as a required plugin during installation.  

Also, in a somewhat over-engineered way, this PR allows a component wrapped in a `withWizardScreen` HOC to render the wizard button. 

### How to test the changes in this Pull Request:

1. Install the plugin on a fresh WP site
2. Complete the first step (Settings) and verify the changes are saved: in Customizer, Reader Revenue wizard, and Yoast settings
3. Observe the site title in browser tab updating on "Site Title" input 😎

### Other information:
